spec/unit/puppet/provider/jenkins_credentials/cli_spec.rb

```diff
@@ -284,74 +254,6 @@
         expect(provider.public_send(k.to_sym)).to eq cred[k].nil? ? :undef : cred[k]
       end
 
-      %w[
-        username
-        password
-        private_key
-        passphrase
-        source
-        key_store_impl
-        content
-        file_name
-        secret_key
-        access_key
-        email_address
-        p12_key
-        json_key
-      ].each do |k|
-        expect(provider.public_send(k.to_sym)).to eq :absent
-      end
-    end
-  end
-
-  shared_examples 'a provider from example hash 7' do
-    it do
-      cred = credentials[5]
```

Comment:
Fun fact: this has a nice copy paste error: hash 7 maps to entry 5 instead of 6. It never showed because these samples don't run.